### DISCUSSION TITLE
Порядок значений ширины и долготы в массиве координат

### DISCUSSION THREAD

**Discussion by Ivanov-Igor-V** [yandex-maps-unofficial/vue-yandex-maps#240] [Q&A (2.x)]
https://codesandbox.io/p/devbox/thirsty-kilby-t9cy6f?file=%2Fsrc%2FMap.vue%3A9%2C5-9%2C19
Если мы возьмем координаты Москвы из этой песочницы и введем им на сайте яндекс карт, мы окажемся не в Москве, а в Голестане. Это умышленно сделано?

**Comment by daniluk4000** [comment]
Добрый день. Процитирую Яндекс: GeoJSON also uses this order https://tools.ietf.org/html/rfc7946#appendix-A.1

Так что вопросы к ним, пожалуйста) vue-yandex-maps порядок координат не трогает, его нам задал Яндекс
